### PR TITLE
Disconnected cluster upgrade improvements for 4.5 RNs

### PR DESCRIPTION
**Pull request by codyhoag**

### CODE REVIEW

**Comment by codyhoag** [comment]
@LalatenduMohanty can you confirm this release note for the disconnected cluster upgrade improvements? I'll update the link here when the docs are available, which I'm tracking in your PR: #21993

**Comment by codyhoag** [comment]
@LalatenduMohanty bump. Can you verify this release note for disconnected cluster upgrade improvements? Thanks!

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -87,6 +87,27 @@ development, and testing.
...
 xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installation-three-node-cluster_installing-bare-metal[Running a three-node cluster].
 
+[id="ocp-4-5-disconnected-cluster-upgrade-improvements"]
+==== Disconnected cluster upgrade improvements
+
+During the upgrade process for a disconnected cluster, the Cluster Version
+Operator (CVO) now downloads the release image for a specific {product-title}
+version and verifies whether the image has been compromised. This provides a way
+to download the signature and provide it to the CVO for image verification in a
+restricted network environment.
```
Comment (LalatenduMohanty) [inline release_notes/ocp-4-5-release-notes.adoc:97]: > the Cluster Version Operator (CVO) now downloads the release image for a specific {product-title}
version and verifies whether the image has been compromised

It was always the case. However previously CVO could only verfiy the images if it has access to `mirror.openshift.com` or to `https://storage.googleapis.com/openshift-release/official/signatures`. Hence customers had to use `--force` while upgrading disconnected environments. 

So as part of 4.5 we have added feature where CVO can verify the release payload if the image signature is available as a ConfigMap. So customers do not need to `--force` anymore.

Comment (LalatenduMohanty) [reply release_notes/ocp-4-5-release-notes.adoc:97]: Suggestion for the above text

```
CVO can now verify the release images if the image signature is available as a ConfigMap in the cluster during the upgrade process for a disconnected cluster.  This removes the need for using `--force` flag while doing upgrades in disconnected environment. 
```

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-5-release-notes.adoc [hunk 2]
```diff
@@ -87,6 +87,27 @@ development, and testing.
 For more information, see
 xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installation-three-node-cluster_installing-bare-metal[Running a three-node cluster].
 
+[id="ocp-4-5-disconnected-cluster-upgrade-improvements"]
+==== Disconnected cluster upgrade improvements
```
Comment (kalexand-rh) [inline release_notes/ocp-4-5-release-notes.adoc:91]: s/Disconnected/Restricted network

File: release_notes/ocp-4-5-release-notes.adoc [hunk 3]
```diff
@@ -87,6 +87,27 @@ development, and testing.
...
+This improved upgrade workflow is completed by running the enhanced
+`oc adm release mirror` command. The following actions are performed:
+
+* Pulls the image signature from the release during the mirroring process.
+* Applies the signature ConfigMap directly to the connected cluster.
+
+Note that the `--force` flag is no longer necessary.
+
+//For more information, see
+//../updating/updating-disconnected-cluster.adoc#updating-restricted-network-cluster[Updating a restricted network cluster]
```
Comment (kalexand-rh) [inline release_notes/ocp-4-5-release-notes.adoc:108]: ./updating/updating-restricted-network-cluster.adoc#updating-restricted-network-cluster[Updating a restricted network cluster]

**Pull request merged**